<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>writer</class>
 <widget class="QMainWindow" name="writer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>784</width>
         <height>465</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" rowspan="2" colspan="2">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QComboBox" name="comboBox">
            <property name="maxVisibleItems">
             <number>15</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QComboBox" name="comboBox_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>80</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QFontComboBox" name="fontComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>170</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>488</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QTextEdit" name="textEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>655</width>
          <height>14</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="actionImport"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuPage">
     <property name="title">
      <string>Page</string>
     </property>
     <addaction name="actionDraw"/>
     <addaction name="actionPagesize"/>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPast"/>
    <addaction name="actionSellect"/>
    <addaction name="separator"/>
    <addaction name="menuPage"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionTool_bar"/>
    <addaction name="actionSide_bar"/>
    <addaction name="actionRuler"/>
   </widget>
   <widget class="QMenu" name="menuInsert">
    <property name="title">
     <string>Insert</string>
    </property>
    <addaction name="actionTable"/>
    <addaction name="actionImage"/>
    <addaction name="actionStyles"/>
    <addaction name="actionMedia"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Us"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuInsert"/>
   <addaction name="menuFormat"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFont"/>
   <addaction name="actionColor"/>
   <addaction name="actionBackground_color"/>
   <addaction name="actionBold"/>
   <addaction name="actionUnderline"/>
   <addaction name="actionI"/>
   <addaction name="actionLeft"/>
   <addaction name="actionRight"/>
   <addaction name="actionCenter"/>
   <addaction name="actionImgae"/>
   <addaction name="actionzoom_in"/>
   <addaction name="actionzoom_out"/>
   <addaction name="actionPDF"/>
   <addaction name="actionLB_Writer"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import </string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPast">
   <property name="text">
    <string>Past</string>
   </property>
  </action>
  <action name="actionSellect">
   <property name="text">
    <string>Sellect</string>
   </property>
  </action>
  <action name="actionTool_bar">
   <property name="text">
    <string>Tool bar</string>
   </property>
  </action>
  <action name="actionSide_bar">
   <property name="text">
    <string>Side bar</string>
   </property>
  </action>
  <action name="actionRuler">
   <property name="text">
    <string>Ruler</string>
   </property>
  </action>
  <action name="actionTable">
   <property name="text">
    <string>Table</string>
   </property>
  </action>
  <action name="actionImage">
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="actionStyles">
   <property name="text">
    <string>Styles</string>
   </property>
  </action>
  <action name="actionMedia">
   <property name="text">
    <string>Media</string>
   </property>
  </action>
  <action name="actionAbout_Us">
   <property name="text">
    <string>About Us</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/font.svg</normaloff>:/font.svg</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/text.svg</normaloff>:/text.svg</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionBackground_color">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/background.svg</normaloff>:/background.svg</iconset>
   </property>
   <property name="text">
    <string>Background color</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/bold.svg</normaloff>:/bold.svg</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionI">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/italic.svg</normaloff>:/italic.svg</iconset>
   </property>
   <property name="text">
    <string>Ittalic</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionLeft">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/text (1).svg</normaloff>:/text (1).svg</iconset>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/alignment.svg</normaloff>:/alignment.svg</iconset>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/align.svg</normaloff>:/align.svg</iconset>
   </property>
   <property name="text">
    <string>Center</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/underline.svg</normaloff>:/underline.svg</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
  </action>
  <action name="actionImgae">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/content.svg</normaloff>:/content.svg</iconset>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="actionDraw">
   <property name="text">
    <string>Draw</string>
   </property>
  </action>
  <action name="actionzoom_in">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Downloads/zoomin.svg</normaloff>:/Downloads/zoomin.svg</iconset>
   </property>
   <property name="text">
    <string>zoom in</string>
   </property>
  </action>
  <action name="actionzoom_out">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Downloads/search.svg</normaloff>:/Downloads/search.svg</iconset>
   </property>
   <property name="text">
    <string>zoom out</string>
   </property>
  </action>
  <action name="actionPagesize">
   <property name="text">
    <string>Pagesize</string>
   </property>
  </action>
  <action name="actionPDF">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/pdf.svg</normaloff>:/pdf.svg</iconset>
   </property>
   <property name="text">
    <string>PDF</string>
   </property>
  </action>
  <action name="actionLB_Writer">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/writer.svg</normaloff>:/writer.svg</iconset>
   </property>
   <property name="text">
    <string>LB Writer</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
